### PR TITLE
Add otuput resolution

### PR DESCRIPTION
**Pull request by ntadimeti**

### CODE REVIEW

File: scripts/cmd_args.py [hunk 1]
```diff
@@ -239,6 +239,16 @@ def add_inference_options(parser):
                help='prefix for the inference result files.')
     parser.add('--deletebg', action='store_true',
                help='delete output bedGraph file')
+    parser.add('--out_resolution', required=False,
+               type=type_or_none_fn(int),
+               help='resolution of output files. default 1bp. \
+                     Atacworks always denoises at 1 base pair \
+                     resolution. If out_resolution is 5, then \
+                     the coverage values for every 5 rows is \
```
Comment (avantikalal) [inline scripts/cmd_args.py:247]: every 5 rows -> every 5 base pairs

File: scripts/cmd_args.py [hunk 2]
```diff
@@ -239,6 +239,16 @@ def add_inference_options(parser):
...
+                     averaged and that becomes the new coverage \
+                     value for every 5 rows. Keep in mind that the \
```
Comment (avantikalal) [inline scripts/cmd_args.py:249]: Delete "and that becomes the new coverage value for every 5 rows"

File: scripts/cmd_args.py (hunk 1 above)
Comment (ntadimeti) [reply scripts/cmd_args.py]: done.

File: scripts/cmd_args.py (hunk 2 above)
Comment (ntadimeti) [reply scripts/cmd_args.py]: done.

**Pull request merged**